redbot/cogs/mod/settings.py

```diff
@@ -283,33 +296,97 @@ async def mentionspam_ban(self, ctx: commands.Context, max_mentions: int):
 
     @modset.command()
     @commands.guild_only()
-    async def deleterepeats(self, ctx: commands.Context, repeats: int = None):
+    async def deleterepeats(self, ctx: commands.Context, repeats: int = None, *channels):
         """Enable auto-deletion of repeated messages.
 
         Must be between 2 and 20.
 
         Set to -1 to disable this feature.
+
+        Add channels if you want to custom the value for the channels (#name_of_channel)
+
+        Set to 0 if you want to use the default value for the channel selected
```

Comment:
```suggestion
        Channel specific overrides can be created by using this command with a passed in value for `channels`.

        If one or more channels are provided, `repeats` can be set to 0 to make those channels use the server default.
```
I don't think this is perfect, but it's at least a bit more descriptive than what is currently there.